rewrite-java-11/src/test/kotlin/org/openrewrite/java/tree/LocalInferredVariableTest.kt

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright 2020 the original author or authors.
+ * <p>
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ * https://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.openrewrite.java.tree
+
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.Assertions.assertTrue
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.extension.ExtendWith
+import org.openrewrite.java.*
+import org.openrewrite.java.JavaTreeTest.NestingLevel.Block
+import org.openrewrite.java.JavaTreeTest.NestingLevel.Class
+
+@ExtendWith(JavaParserResolver::class)
+class VariableDeclarations11Test : JavaTreeTest {
+    fun javaParser(): Java11Parser.Builder = Java11Parser.builder()
+
+    @Test
+    fun implicitlyDeclaredLocalVariable(jp: JavaParser) {
+
+        assertParsePrintAndProcess(
+            jp, Block, """
+                var a = "";
+                var/* comment */b = "";
+                /*comment*/var c = "";
+                var     d = "";
+                long /* yep */ i /* comments */, /*everywhere*/ j; 
+            """
+        )
+    }
```

Comment:
It would be stylistically consistent with our other tests to use the single-expression form where possible:
```suggestion
    @Test
    fun implicitlyDeclaredLocalVariable(jp: JavaParser) = assertParsePrintAndProcess(
            jp,
            Block,
            """
                var a = "";
                var/* comment */b = "";
                /*comment*/var c = "";
                var     d = "";
                long /* yep */ i /* comments */, /*everywhere*/ j; 
            """
        )
```

Reply:
hah..this was an artifact of having a conditional check for the parser's supported language version, I will fix this.

Reply:
done